addons/project/views/project_update_views.xml

```diff
@@ -53,7 +53,7 @@
                         </page>
                     </notebook>
                 </sheet>
-                <div class="oe_chatter">
+                <div class="oe_chatter d-print-none">
```

Comment:
I am not sure to understand why it is a improve as you explain in your commit title. For me, it is more to avoid printing the chatter, no?

Reply:
> I am not sure to understand why it is a improve as you explain in your commit title. For me, it is more to avoid printing the chatter, no?

I have updated the commit message as well as uploaded the screenshot  in  the PR description.thank you